package/nginx.conf

```diff
@@ -0,0 +1,17 @@
+events {}
```

Comment:
purpose of adding this here is to be able to do local testing, I'll probably add some stuff to run the docker command and curl some stuff for smoke tests to CI when I get to it